src/WinRPM.jl

```diff
@@ -1,3 +1,5 @@
+VERSION >= v"0.4.0" && __precompile__(true)
```

Comment:
this is not specific enough on the version number. use `isdefined(Base, :__precompile__)` instead